homework/web_scrapping1.py

```diff
@@ -0,0 +1,38 @@
+import requests
+from bs4 import BeautifulSoup 
+import sqlite3
+
+conn= sqlite3.connect('cputest.db')
```

Comment:
@sampleharvgithub Any reason for having the database name as "cputest" instead of a generic name which would indicate the purpose/data contained in that database?